index.js

```diff
@@ -0,0 +1,180 @@
+    <div class="desktop-popup">
+    <div class="desktop-inner">
+    <img class="desktop-innerimg" src="/Assets/Images/ICancel.png" alt="SnapshootPortfolioDesktop">
+    <img class="project-img" src="/Assets/Images/SnapshootPortfolioDesktop.png" alt="SnapshootPortfolioDesktop">
+    <img class="projectimgmob" src="/Assets/Images/SnapshootPortfolio.png" alt="SnapshootPortfolioDesktop">
+   
+    <div class="headbuttons">
+      <div class='headerfullwrapper'>
+        <h3 class="itemheader">Keeping track of hundreds of components</h3>
+        <ul class='languages-used'>
+        <li class='languages-used-li'>Ruby on Rails</li>
+        <li class='languages-used-li'>css</li>
+        <li class='languages-used-li'>html</li>
+        </ul>
+      </div>
+
+      <span class='mobile-tag'>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the release</span>
+      <div class="btnwrappers">
+        <a href="#" id="btnwrappers-a" class="see">See Live <img src='../Assets/Images/seeliveicon.png' alt="hello" /></a>
+        <a href="#" id="btnwrappers-a" class="see">See Source <img src='../Assets/Images/Vector.png' alt="hello" /></a>
+      </div>
+    </div>
+    <div class="listech">
+      <ul>
+        <li>codekit</li>
+        <li>Github</li>
+        <li>Javascript</li>
+        <li>Bootstrap</li>
+        <li>Terminal</li>
+        <li>codepen</li>
+      </ul>
+      <span>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea</span>
+    </div>
+
+    </div>
+    `;
+
+  const btnsee=document.querySelector('.popupContainer')
+  btnsee.appendChild(pop)
+  btnsee.classList.remove('popupContainer')
+  btnsee.classList.add('popout-container')
+  btnsee.style.display='none'
+  const btnclose=document.querySelector('.p-image')
+const deskclose=document.querySelector('.desktop-innerimg')
+const mobileclose=document.querySelector('.projectimgmob')
+const btx=document.querySelectorAll('#seeproject').forEach((project) => {
+    deskclose.onclick=()=>{
+      btnsee.style.display='none' 
+    } 
+    mobileclose.onclick=()=>{
+      btnsee.style.display='none' 
+    }
+  project.onclick=()=>{
+    btnsee.style.display='flex'
+  }
```

Comment:
I think you did a great job with the email validation. However, there is a tiny issue with your mobile version. The mobile version is not doing the validation. Kindly fix this issue :pray: :pray: